### ISSUE TITLE
Typos in AZURE_STORAGE_ACCOUNT

### ISSUE THREAD

**Issue by itpropro** [Azure/azure-storage-fuse#410]
I think AZURE_STORAGE_ACCOUNT in blobfuse.cpp should be spelled with two "C" and there should be an empty space " " before "environment".
The current error message looks like this:

```bash
[...] No config file was specified and the AZURE_STORAGE_ACCCOUNTenvironment variable was empty
```

https://github.com/Azure/azure-storage-fuse/blob/ff8799a49acabb65ab5b7baf0f36dba78fc6f136/blobfuse/blobfuse.cpp#L167

**Comment by NaraVen** [comment]
Thank you. This is a typo. We will push the change soon. I will close this when the PR is done.

**Comment by NaraVen** [comment]
This is fixed in release 1.2.3
